lib/tasks/manage.rake

```diff
@@ -23,6 +23,50 @@ namespace :manage do
     end
   end
 
+  def move_to_new_generic_business_finder_emails
+    email_subject = "Changes to GOV.UK email alerts"
+    email_utm_parameters = {
+      'utm_source' => email_subject,
+      'utm_medium' => 'email',
+      'utm_campaign' => 'govuk-subscription-ended'
+      }
+    email_redirect = EmailTemplateContext.new.add_utm('https://gov.uk/email/manage', email_utm_parameters)
+
+    bulk_business_move_template = <<~BODY.freeze
+      Hello,
+
+      You've subscribed to get email alerts about EU Exit guidance for your business.
+
+      GOV.UK is changing the type of email alerts you get to make sure you find out when any EU Exit business guidance is added or updated.
+
+      You can [change how often you get updates or unsubscribe](<%= email_redirect %>) from GOV.UK email alerts.
```

Comment:
Could we use `#{email_redirect}` here instead of needing to use `ERB`?

Reply:
Done!